dub.json

```diff
@@ -13,30 +13,229 @@
         "Sebastián E. Peyrott",
         "John Reimer",
         "Mike Wey",
-        "hauptmech"
+        "hauptmech",
+        "KonstantIMP"
     ],
-    "dependencies": {
-        "gtk-d:gtkd": "*",
-        "gtk-d:gstreamer": "*"
-    },
-
-    "subPackages" : [
-        {
-            "name": "gtkd",
-            "targetType": "library",
-            "targetName": "gtkd-3",
-            "sourcePaths": ["generated/gtkd"],
-            "libs-posix": ["dl"],
-            "importPaths": ["generated/gtkd"]
-        },
-        {
-            "name": "gstreamer",
-            "targetType": "library",
-            "targetName": "gstreamerd-3",
-            "sourcePaths": ["generated/gstreamer"],
-            "libs-posix": ["dl"],
-            "importPaths": ["generated/gstreamer"],
-            "dependencies": {"gtk-d:gtkd":"*"}
-        }
-    ]
+    "subPackages": [
```

Comment:
With how this is now structured, wouldn't it be better to separate things out into different repositories?

Like whats already been done for GLib: https://github.com/gtkd-developers/GlibD

Reply:
for the core packages instead of different repositories I would recommend using subpackages in separate folders (useful to install all the common stuff at once)

Reply:
> With how this is now structured, wouldn't it be better to separate things out into different repositories?
> 
> Like whats already been done for GLib: https://github.com/gtkd-developers/GlibD

It will be harder to update packages I think.